reporter/osm.py

```diff
@@ -230,7 +230,76 @@ def extract_buildings_shapefile(
         qgis_version)
 
 
-def extract_roads_shapefile(file_path, qgis_version=1, output_prefix=''):
+def extract_buildings_points_shapefile(file_path, qgis_version=2, output_prefix=''):
+    """Convert the OSM xml file to a buildings points shapefile.
+
+    This is a multistep process:
+        * Create a temporary postgis database
+        * Load the osm dataset into POSTGIS with osm2pgsql and our custom
+             style file.
+        * Save the data out again to a shapefile
```

Comment:
Above this line you should add :
* Calculate point on surface for building centroids